HoloJS/HoloJsHost/HologramJS.cpp

```diff
@@ -155,16 +151,23 @@ HologramScriptHost::RunWebScriptApp(
 		API::ImageElement::BaseUrl.assign(basePath);
 		API::VideoElement::BaseUrl.assign(basePath);
 
-		for (const auto& script : loader->LoadedScripts())
-		{
-			JsValueRef result;
-			RETURN_IF_JS_ERROR(JsRunScript(script.c_str(), m_jsSourceContext++, L"", &result));
-		}
+		ExecuteScripts(loader);
+
 	}
 
 	return true;
 }
 
+void 
+HologramScriptHost::ExecuteScripts(std::unique_ptr<HologramJS::ScriptsLoader>& loader) 
```

Comment:
Maybe this should be a method of the ScriptsLoader; that way there is no need to expose the loaded scripts, just the Execute method to execute them all.